app/services/organizations/delete.rb

```diff
@@ -15,5 +18,24 @@ def self.call(...)
     private
 
     attr_reader :org
+    attr_accessor :article_ids
+
+    def delete_notifications
+      sql = <<-SQL.squish
```

Comment:
At first I though you could rewrite this as

```ruby
Notification.delete_by(id: Notification.select(:id).where(organization_id: org.id))
```
but `BulkSqlDelete` requires a SQL string.